### ISSUE TITLE
won't work on windows.

### ISSUE THREAD

**Issue by TroyWorks** [mikenorthorp/gulp-shopify-upload#2]
C:\Timber-2.0.2>npm install gulp-shopify-upload
npm WARN package.json jsxc@0.9.0 No repository field.
npm http GET https://registry.npmjs.org/gulp-shopify-upload
npm http 304 https://registry.npmjs.org/gulp-shopify-upload
npm http GET https://registry.npmjs.org/isbinaryfile/2.0.0
npm http GET https://registry.npmjs.org/async/0.2.9
npm http GET https://registry.npmjs.org/shopify-api/0.2.2
npm http GET https://registry.npmjs.org/path/0.11.14
npm http GET https://registry.npmjs.org/through2/0.6.3
npm http GET https://registry.npmjs.org/pwuid/1.0.1
npm http GET https://registry.npmjs.org/gulp-util/3.0.2
npm http 304 https://registry.npmjs.org/async/0.2.9
npm http 304 https://registry.npmjs.org/shopify-api/0.2.2
npm http 304 https://registry.npmjs.org/pwuid/1.0.1
npm http 304 https://registry.npmjs.org/through2/0.6.3
npm http 304 https://registry.npmjs.org/isbinaryfile/2.0.0
npm http 304 https://registry.npmjs.org/gulp-util/3.0.2
npm http 304 https://registry.npmjs.org/path/0.11.14
npm ERR! Error: ENOENT, lstat 'C:...\node_modules\gulp-shopify-upload\node_modules\isbinaryfile\package.jso
n'
npm ERR! If you need help, you may report this _entire_ log,
npm ERR! including the npm and node versions, at:
npm ERR!     http://github.com/npm/npm/issues

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\Program Files\nodejs\\node.exe" "C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js"
 "install" "gulp-shopify-upload"
npm ERR! cwd C:\Timber-2.0.2
npm ERR! node -v v0.10.26
npm ERR! npm -v 1.4.3
npm ERR! path C:...\node_modules\gulp-shopify-upload\node_modules\isbinaryfile\package.json
npm ERR! fstream_path C:...\node_modules\gulp-shopify-upload\node_modules\isbinaryfile\package.json
npm ERR! fstream_type File
npm ERR! fstream_class FileWriter
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! fstream_stack C:\Program Files\nodejs\node_modules\npm\node_modules\fstream\lib\writer.js:284:26
npm ERR! fstream_stack Object.oncomplete (fs.js:107:15)
npm ERR! EEXIST, mkdir 'C:...\node_modules\gulp-shopify-upload\node_modules\async'
File exists: C:...\node_modules\gulp-shopify-upload\node_modules\async
Move it away, and try again.

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\Program Files\nodejs\\node.exe" "C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js"
 "install" "gulp-shopify-upload"
npm ERR! cwd C:...\IdeaProjects\ShopifyTeeTheme\src\Timber-2.0.2
npm ERR! node -v v0.10.26
npm ERR! npm -v 1.4.3
npm ERR! path C:...\node_modules\gulp-shopify-upload\node_modules\async
npm ERR! fstream_path C:...\node_modules\gulp-shopify-upload\node_modules\async\README.md
npm ERR! fstream_type File
npm ERR! fstream_class FileWriter
npm ERR! code EEXIST
npm ERR! errno 47
npm ERR! fstream_stack C:\Program Files\nodejs\node_modules\npm\node_modules\fstream\lib\writer.js:171:23
npm ERR! fstream_stack C:\Program Files\nodejs\node_modules\npm\node_modules\mkdirp\index.js:37:53
npm ERR! fstream_stack Object.oncomplete (fs.js:107:15)
npm ERR! EEXIST, mkdir 'C:...\node_modules\gulp-shopify-upload\node_modules\through2'
File exists: C:...\node_modules\gulp-shopify-upload\node_modules\through2
Move it away, and try again.

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\Program Files\nodejs\\node.exe" "C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js"
 "install" "gulp-shopify-upload"
npm ERR! cwd C:...\IdeaProjects\ShopifyTeeTheme\src\Timber-2.0.2
npm ERR! node -v v0.10.26
npm ERR! npm -v 1.4.3
npm ERR! path C:...\node_modules\gulp-shopify-upload\node_modules\through2
npm ERR! fstream_path C:...\node_modules\gulp-shopify-upload\node_modules\through2.npmignore
npm ERR! fstream_type File
npm ERR! fstream_class FileWriter
npm ERR! code EEXIST
npm ERR! errno 47
npm ERR! fstream_stack C:\Program Files\nodejs\node_modules\npm\node_modules\fstream\lib\writer.js:171:23
npm ERR! fstream_stack C:\Program Files\nodejs\node_modules\npm\node_modules\mkdirp\index.js:37:53

**Comment by mathiasvalentin** [comment]
I'm getting:
npm ERR! notsup Unsupported
npm ERR! notsup Not compatible with your operating system or architecture: pwuid@1.0.1
npm ERR! notsup Valid OS:    !win32
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   win32
npm ERR! notsup Actual Arch: x64

**Comment by mikenorthorp** [comment]
Don't plan on supporting it for windows at this time, if you feel like doing a pull request to make it work on windows, go for it and I can merge it in if it works.

**Comment by mikedidthis** [comment]
I managed to resolve this issue:

1) Remove `"gulp-shopify-upload": "1.0.10"` from `devDependecies` in projects `package.json`.
2) Run `npm install` from projects root.
2) Download repo as a zip.
3) Extract zip to project root, under: `node_modules/gulp-shopify-upload`.
4) Remove `"pwuid": "1.0.2"` from `dependecies` in plugins `package.json`.
5) Run `npm install` from plugins root.
6) Add `"gulp-shopify-upload": "1.0.10"`to `devDependecies` in the projects `package.json`. (For sanity).
7) Run `gulp` from the projects root. 

I can't find `pwuid` being used anywhere, but I only glanced. Happy to make the changes if this works for others and doesn't break anything.

> tl;dr - I managed to fix this issue by removing `"pwuid": "1.0.2"` from dependencies
